docs/pages/attributesJSON.rst

```diff
@@ -46,9 +46,125 @@ Below are the supported JSON tags for Physics Manager Attributes templates, and
+
+"default_attributes":
+  - JSON object
+  - This tag references the default configuration values to be used for any attributes' values not explicitly specified for whatever type of construction (i.e. "stages", "objects", etc.) is being referenced. See the individual attributes types' documentation for supported tags and format.
+
+"paths":
+  - JSON object
+  - This tag references an object consisting of key-value pairs where each key is the exension of a particular config (".json") or (for "objects" and "stages" only) a valid asset (such as ".glb"), and the value is a list to the paths to query for the desired configuration files or assets.  For asset-related k-v pairs, an attributes will be generated for each asset found, using any specified defaults provided, with the render and collision asset handles set to the asset's name.
+
+"configs":
+  - array of JSON objects
+  - This array provides the user with an easy interface to override or extend any existing configurations.  Each array element is a JSON object that supports the creation of customized attributes.  Either "original_file" or "template_handle" (or both) must be specified.  The following tags are supported in each individual JSON object:
+
+    "original_file":
+    - string
+    - If provided, this tag is the qualified filename of an existing attributes to use as a baseline attributes. Either this tag or "template_handle" must be specified for each JSON object (although both can also be given)
+
+    "template_handle"
+    - string
+    - If provided, this is the handle to use to store the new attributes in the Scene Dataset library; if not provided, any customizations will be stored under the "original_file"-specified name. Either this tag or "original_file" must be specified for each JSON object (although both can also be given)
+
+    "attributes"
+    - JSON object
+    - This tag references a configuration appropriate for the construction type. If "original_file" is specified, all the values from "original_file" are used, and overridden by any specified in this tag.  If "template_handle" is specified, the new attributes are saved with the specified "template_handle", otherwise they are saved with the "original_file" handle. See the individual attributes types' documentation for supported tags and format.
+
+'Scene Dataset: Navmeshes and Semantic Scene Descriptors'
+---------------------------------------------------------
+
+The Scene Dataset Attributes configuration also contains references to the navmeshes, using the JSON tag "navmesh_instances", and semantic scene descriptor files, using the tag "semantic_scene_descriptor_instances", that are available to the Scene Dataset.  These tags each reference a JSON object consisting of key-value pairs where the keys are the aliases to use in scene instances for a particular navmesh or semantic scene descriptor, and the values are the paths to the navmesh or SSD file, respectively.
```

Comment:
We should hyperlink Navmeshes to the colab explaining Navmeshes. 

And what is a semantic scene descriptor file?

Reply:
> And what is a semantic scene descriptor file?

The "house" file. Alex and I felt that this was more descriptive of its functionality.